lib/vnet/user_process_windows.go

```diff
@@ -42,36 +46,124 @@ func runPlatformUserProcess(ctx context.Context, config *UserProcessConfig) (pm
 		}
 	}()
 
-	listener, err := net.Listen("tcp", ":0")
+	ipcCreds, err := newIPCCredentials()
+	if err != nil {
+		return nil, trace.Wrap(err, "creating credentials for IPC")
+	}
+	serverTLSConfig, err := ipcCreds.server.serverTLSConfig()
+	if err != nil {
+		return nil, trace.Wrap(err, "generating gRPC server TLS config")
+	}
+	credDir, err := os.MkdirTemp("", "vnet_service_certs")
```

Comment:
Do I correctly understand that it's the user process that creates the directory, but it's the admin process that removes it? Can we keep the responsibility in the same place?

Reply:
I added Standard Delete to the user process ACL and now the user process cleans up the files